tdei-api-gateway.json

```diff
@@ -687,21 +678,21 @@
                 ],
                 "responses": {
                     "200": {
-                        "description": "Returns status of record",
+                        "description": " Status of the formatting Job ",
                         "content": {
                             "application/json": {
                                 "schema": {
-                                    "$ref": "#/components/schemas/RecordStatus"
+                                    "$ref": "#/components/schemas/OSWFormatStatusResponse"
```

Comment:
how does one download the reformatted output?

Reply:
@susrisha , fyi

Reply:
Added ticket for the conversion job. Decided to give out another API for downloading the converted files. Will be implemented separately